options_compaction.go

```diff
@@ -33,6 +33,8 @@ const (
 // CompactRangeOptions represent all of the available options for compact range.
 type CompactRangeOptions struct {
 	c *C.rocksdb_compactoptions_t
+
+	fullHistoryTsLow []byte
```

Comment:
How about removing this field. Seem like we do not need to track this. Its invoker (caller) responsibility